demo.py

```diff
@@ -0,0 +1,79 @@
+import argparse
+import numpy
+
+import torch
+import cv2
+from lib.config import Config
+import os
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description="Train lane detector")
+    parser.add_argument(
+        "mode", choices=["video", "image"], help="Video or image?")
+    parser.add_argument("-path", help="image/video path", required=True)
+    parser.add_argument("-cfg", help="Config file", required=True)
+    args = parser.parse_args()
+    return args
+
+
+def process(model, image, img_w, img_h, cfg, device):
+    image = cv2.resize(image, (img_w, img_h))

```

Comment:
You should use the `NoLabelDataset` class so that you don't need to repeat the code used to draw predictions nor the steps required to transform raw images to input tensors.